src/blocking/spi.rs

```diff
@@ -104,3 +104,26 @@ pub mod write_iter {
         }
     }
 }
+
+/// Operation for transactional SPI trait
+///
+/// This allows composition of SPI operations into a single bus transaction
+#[cfg(feature = "unproven")]
+#[derive(Debug, PartialEq)]
+pub enum Operation<'a, W: 'static> {
+    /// Write data from the provided buffer, discarding read data
+    Write(&'a [W]),
+    /// Write data out while reading data into the provided buffer
+    Transfer(&'a mut [W]),
+}
+
+/// Transactional trait allows multiple actions to be executed
+/// as part of a single SPI transaction
+#[cfg(feature = "unproven")]
```

Comment:
```suggestion
```

Reply:
I thought we weren't doing `unproven` anymore. ;)

Reply:
good catch, thanks. these PRs predate this change by some months 🤣